package.json

```diff
@@ -116,6 +116,7 @@
     "@storybook/addons": "^6.2.9",
     "@storybook/core": "^6.2.9",
     "@storybook/react": "^6.2.9",
+    "@styled-system/should-forward-prop": "^5.1.5",
```

Comment:
Can we move this dependency to the package.json file in the `packages/paste-libraries/styling` folder please? 

We also pin library dependencies to make sure no odd behavior happens downstream, so feel free to remove the `^`.

Reply:
Hey @TheSisb , I moved this dependency in `styling` package and I fixed the version.